caselaw.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Caselaw Access Project</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="robots" content="noindex" />
+		<meta name="author" content="Your name" />
+		<meta name="description" content="Explore American Caselaw" />
+		<meta property="og:title" content="Caselaw Access Project" />
+		<meta property="og:description" content="Explore American Caselaw" />
+		<meta property="og:url" content="https://case.law/" />
+		<meta property="og:type" content="article" />
+		<meta property="og:site_name" content="Caselaw Access Projects" />
+
+		<link href="/css/global.css" rel="stylesheet" />
+		<script type="module" src="/components/cap-map.js"></script>
+		<script type="module" src="/components/cap-header.js"></script>
+		<script type="module" src="/components/cap-stats.js"></script>
+		<script type="module" src="/components/cap-section-highlight.js"></script>
+		<script type="module" src="/components/cap-contact.js"></script>
+		<script type="module" src="/components/cap-footer.js"></script>
+		<script type="module" src="/components/cap-content-router.js"></script>
+		<script type="module" src="/components/cap-case.js"></script>
+		<script type="module" src="/components/cap-volume.js"></script>
+		<script type="module" src="/components/cap-jurisdictions.js"></script>
+		<script type="module" src="/components/cap-reporter.js"></script>
+
+		<script>
+			var BUCKET_ROOT = "https://cap-redacted-demo.s3.us-west-2.amazonaws.com";	
+		</script>
+	</head>
+	<body>
+		<cap-header></cap-header>
```

Comment:
Tiny thing: I renamed `cap-header` to `cap-nav` while working on the about page before the holidays. 

The nav contains a `header` element, but it felt like `cap-header` might be a more fitting name for the reusable headers that interior pages need.